inlong-sdk/transform-sdk/src/test/java/org/apache/inlong/sdk/transform/process/TestTransformStringFunctionsProcessor.java

```diff
@@ -59,6 +59,42 @@ public class TestTransformStringFunctionsProcessor {
         kvSink = new KvSinkInfo("UTF-8", dstFields);
     }
 
+    @Test
+    public void testLowerFunction() throws Exception {
+        String transformSql1 = "select lower(string1) from source";
+        TransformConfig config1 = new TransformConfig(transformSql1);
+        TransformProcessor<String, String> processor1 = TransformProcessor
+                .create(config1, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        // case1: lower(ApPlE)
+        List<String> output1 = processor1.transform("ApPlE|banana|cloud|2|1|3", new HashMap<>());
+        Assert.assertEquals(1, output1.size());
+        Assert.assertEquals(output1.get(0), "result=apple");
+
+        // case2: lower("")
+        List<String> output2 = processor1.transform("|banana|cloud|2|1|3", new HashMap<>());
+        Assert.assertEquals(1, output2.size());
+        Assert.assertEquals(output2.get(0), "result=");
+    }
+
+    @Test
+    public void testUpperFunction() throws Exception {
+        String transformSql1 = "select upper(string1) from source";
+        TransformConfig config1 = new TransformConfig(transformSql1);
+        TransformProcessor<String, String> processor1 = TransformProcessor
+                .create(config1, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        // case1: lower(ApPlE)
+        List<String> output1 = processor1.transform("ApPlE|banana|cloud|2|1|3", new HashMap<>());
+        Assert.assertEquals(1, output1.size());
+        Assert.assertEquals(output1.get(0), "result=APPLE");
+
+        // case2: lower("")
+        List<String> output2 = processor1.transform("|banana|cloud|2|1|3", new HashMap<>());
+        Assert.assertEquals(1, output2.size());
+        Assert.assertEquals(output2.get(0), "result=");
+    }
```

Comment:
It is better to add a test case: `lower(NULL)`

Reply:
Thank you for your suggestion. We have modified the null handling logic and added relevant test cases.